moveit_py/src/moveit/moveit_core/planning_scene/planning_scene.h

```diff
@@ -57,6 +57,12 @@ Eigen::MatrixXd getFrameTransform(std::shared_ptr<planning_scene::PlanningScene>
 
 moveit_msgs::msg::PlanningScene getPlanningSceneMsg(std::shared_ptr<planning_scene::PlanningScene>& planning_scene);
 
+bool saveGeometryToFile(std::shared_ptr<planning_scene::PlanningScene>& planning_scene,
+                        const std::string& file_path_and_name);
+
+bool loadGeometryFromFile(std::shared_ptr<planning_scene::PlanningScene>& planning_scene,
+                          const std::string& file_path_and_name);
+
```

Comment:
There is no need to export these methods. They are only used within the wrapper.
To emphasize that point, you could also enclose them into an anonymous namespace there.